tests/testapp/flows.py

```diff
@@ -13,3 +13,14 @@ class SavableFlow(Flow):
         .Assign(username='admin') \
         .Next(this.end)
     end = flow.End()
+
+
+class TestFlow(Flow):
+    process_cls = SavableProcess
+    start = flow.Start(StartProcessView, fields=[]).Next(this.savable_task)
+    savable_task = flow.View(SavableProcessView) \
```

Comment:
split has attribute `branches`, same as switch, and join is like a regular task, it has attribure `_next`. There are 3 types in this sense, most of the tasks have `_next`, some have `branches`,and  `flow.If` has `_on_true` and `_on_false`.